### PR TITLE
Eugens suggested edits

### PR DESCRIPTION
**Pull request by mcgonagle**
*Issue #, if available:*

*Description of changes:*
Incorporated Eugene's suggestions

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/050_module_1/55_create_the_load_balancers.md [hunk 1]
```diff
@@ -11,11 +11,16 @@ Now that our Armory Application and Kubernetes Namespaces are created, we're goi
 For each of our environments, we're going to set up two Kubernetes resources:
 
 - A "Service" of type "ClusterIP", which acts as an internal load balancer to access our applications
-- An "Ingress", which will configure Traefik to point specific paths on the Minnaker VM to our internal Services
+- An "Ingress", which acts as an exteranl entry will route the requests to the appropriate backend services
```
Comment (eugenemu) [inline content/050_module_1/55_create_the_load_balancers.md]: external*

File: content/050_module_1/57_run_the_pipeline_with_a_different_parameter.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ weight: 12
...
 
+What this dynamic configuration possible is Armory Enterprises SpEL (Spring Expression Language). The Spring Expression Language (SpEL for short) is a powerful expression language that supports querying and manipulating an object graph at runtime. The language syntax is similar to Unified EL but offers additional features, most notably method invocation and basic string templating functionality. For more information please refer to Armory's SpEL [documentation](https://docs.armory.io/docs/spinnaker-user-guides/expression-language/).
```
Comment (eugenemu) [inline content/050_module_1/57_run_the_pipeline_with_a_different_parameter.md]: "What makes this dynamic configuration possible, is..."

File: content/070_module_3/71_scaleup_and_rollback.md [hunk 3]
```diff
@@ -5,11 +5,11 @@ draft: false
 weight: 41
 ---
 # Scale up and Rollback
-<!--Add some context here-->
+The ability to roll your application forward and back, to be able to provision new pods or roll back to a previous state is an important one. Modern applications are living breathing things and as software is developed and rolled out issues may arise during the initial phase of the release and require fixing. This functionality of rolling back to a good known state is important and critical for any modern software delivery system. Kubernetes provides the ability to scale up known good applications, as well as roll them back to previous versions. This functionality is exposed in Armory Enterprise and is walked through in this next module.
```
Comment (eugenemu) [inline content/070_module_3/71_scaleup_and_rollback.md]: "Modern applications are living breathing entities; as deployment occurs, issues may arise during the initial phase of the release and requires patching"

File: content/070_module_3/71_scaleup_and_rollback.md [hunk 4]
```diff
@@ -5,11 +5,11 @@ draft: false
...
 
-Before we scale up and rollback, we need to deploy the application we will use by creating a pipeline. The simple application that we will deploy displays the fruit name "Apple". We will be deploying a new version of the application that displays the fruit name "Banana". Inorder provide external access to the applications, the deployment will include an Ingress. You will find the ingress endpoint in the Armory Enterprise UI which you will enter in the address field of your browser. In this section you will also update the application by making a siple modification to the yaml. 
+Before we scale up and rollback, we need to deploy the application we will use by creating a pipeline. The simple application that we will deploy displays the fruit name "Apple". We will be deploying a new version of the application that displays the fruit name "Banana". In order provide external access to the applications, the deployment will include an Ingress. You will find the ingress endpoint in the Armory Enterprise UI which you will enter in the address field of your browser. In this section you will also update the application by making a siple modification to the yaml.
```
Comment (eugenemu) [inline content/070_module_3/71_scaleup_and_rollback.md]: In order to provide external access to the applications

File: content/070_module_3/72_rollback.md [hunk 5]
```diff
@@ -7,6 +7,8 @@ weight: 42
 
 ## Rollback
 
+The two key reasons why rollbacks are so important are consistency and available. Since modern applications are composed of many microservices, you need the ability to manage each microservice individually. For example in the update of a microservice application, many services might need to be updated at the same time. A failure or bug in one or more services can cause unexpected behavior in the application or cause it to completely fail. This is why rollback capability is so critical. It allows you to rollback an issue to a consistent state.
```
Comment (eugenemu) [inline content/070_module_3/72_rollback.md]: The two reasons why rollbacks are crucial are: consistency and application availability.

File: content/070_module_3/74_summary.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+title: "4. Summary "
+chapter: true
+draft: false
+weight: 42
+---
+
+## Summary
+In this final module of the workshop you learned about scale up and roll back, two important techniques for modern continuous delivery. Blue/Green as well as Canary rollouts depend on these features. We leveraged Kubernetes for this lab, but rollback and scale up, blue/green and canary are all important and
```
Comment (eugenemu) [inline content/070_module_3/74_summary.md]: "In this final module of the workshop, you've learned how to scale up and roll back a deployment: two important techniques for modern continuous delivery. B/G as well as Canary rollout methods are dependent on these two techniques. Although we leveraged Kubernetes for this lab, these techniques are applicable to whichever deployment target you may use."

File: content/050_module_1/55_create_the_load_balancers.md (hunk 1 above)
Comment (mcgonagle) [reply content/050_module_1/55_create_the_load_balancers.md]: Fixed

File: content/050_module_1/57_run_the_pipeline_with_a_different_parameter.md (hunk 2 above)
Comment (mcgonagle) [reply content/050_module_1/57_run_the_pipeline_with_a_different_parameter.md]: fixed

File: content/070_module_3/71_scaleup_and_rollback.md (hunk 3 above)
Comment (mcgonagle) [reply content/070_module_3/71_scaleup_and_rollback.md]: Fixed

File: content/070_module_3/71_scaleup_and_rollback.md (hunk 4 above)
Comment (mcgonagle) [reply content/070_module_3/71_scaleup_and_rollback.md]: Fixed

File: content/070_module_3/72_rollback.md (hunk 5 above)
Comment (mcgonagle) [reply content/070_module_3/72_rollback.md]: Fixed

File: content/070_module_3/74_summary.md (hunk 6 above)
Comment (mcgonagle) [reply content/070_module_3/74_summary.md]: Fixed

**Review by eugenemu — APPROVED** [review]

**Pull request merged**